services/vidispine/export.go

```diff
@@ -408,3 +407,33 @@ func (c *Client) GetDataForExport(itemVXID string) (*ExportData, error) {
 
 	return &out, nil
 }
+
+// ConvertFromClipTCTimeSpaceToSequenceRelativeTimeSpace ain't this a nice name?
+func ConvertFromClipTCTimeSpaceToSequenceRelativeTimeSpace(clip *Clip, chapter *MetadataResult, tcStart float64) *MetadataResult {
```

Comment:
Amazing function name, but making it private makes it less noticeable :). Not that I wouldn't want to see it of course.